packages/contract/src/__test__/coverage-contract/coverage-contract.test.ts

```diff
@@ -73,9 +75,9 @@ describe('Coverage Contract', () => {
   });
 
   it('should test u64 variable type', async () => {
-    const INPUT = BigInt(RUST_U32_MAX + 1);
+    const INPUT = bn(RUST_U32_MAX).add(bn(1));
     const { value } = await contractInstance.functions.echo_u64(INPUT).call();
-    expect(value).toBe(INPUT);
+    expect(value).toBe(toHex(INPUT));
```

Comment:
it's because both `INPUT` and `value` are gonna be `BN` instances (objects). 

The object instances can be different, but same value. To compare the value for testing purpose we convert to Hex first

we can also do `value.eq(INPUT)`